src/Raven.Client/Documents/Attachments/RetiredAttachmentsConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public sealed class RetiredAttachmentsConfiguration : IDynamicJson
         public Dictionary<string, RetiredAttachmentsDestinationConfiguration> Destinations { get; set; }
         public long? RetireFrequencyInSec { get; set; }
         public long? MaxItemsToProcess { get; set; }
+        public short? ConcurrentUploads { get; set; }
```

Comment:
```suggestion
        public int? ConcurrentUploads { get; set; }
```

What does it means when it is null?

Reply:
it means the user didn't specify `ConcurrentUploads` so we will use the default value